.github/workflows/generate-package-metadata.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         env:
           REPO_SLUG: ${{ matrix.repoSlug }}
       - name: Check if there is already an open PR
-        if: contains(needs.generate-packages-list.outputs.prs, steps.regex-prov.outputs.group2 )  
+        if: contains(needs.generate-packages-list.outputs.prs, "Publish Package Metadata ${{steps.regex-prov.outputs.group2}}" )  
```

Comment:
I wonder if it would make sense to abstract the prefix into a variable in the environment if possible so this will stay consistent with [line 91](https://github.com/pulumi/registry/pull/3739/files#diff-8bc697d3942ce672edeca3685a03be4b144f21c8b2c2dea50fb95cf4d6556d0fR91) and make it a little less prone to error down the road if any of this changes... though if it is not super straight forward to do this or takes up a lot of time, disregard this.